.nvmrc

```diff
@@ -0,0 +1 @@
+10.16
```

Comment:
ch-js is on 12.11.1 should we use the same version here?

Reply:
as discussed ;)